documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "development Copy",
+	"values": [
+		{
+			"key": "email-from",
+			"value": "solutions",
+			"enabled": true
+		},
+		{
+			"key": "email-template-id",
```

Comment:
Is this variable needed/used here? If so - for which endpoint in this simplified collection?

Reply:
Yes, this is used for {{notification-api-url}}/v2/notifications/email endpoint. 

I didn't document it here, but you can see that it's called for in the post body for that endpoint.